http/base/rpc/interfaces.go

```diff
@@ -764,3 +764,30 @@ func GetGrantOng(params []interface{}) map[string]interface{} {
 	}
 	return responseSuccess(rsp)
 }
+
+//get shard chainconfig
+func GetShardChainConfig(params []interface{}) map[string]interface{} {
+	if len(params) < 2 {
+		return responsePack(berr.INVALID_PARAMS, nil)
+	}
+	var shardID common.ShardID
+	var height uint32
+	switch params[0].(type) {
```

Comment:
maybe change to ```if id, ok := params[0].(uint64); ok{...} else { return err}```

Reply:
no,in rpc interface,we can use string or float64,parse param type